### ISSUE TITLE
--local-development flag does not work

### ISSUE THREAD

**Issue by sudermanjr** [FairwindsOps/reckoner#101]
The `--local-development` flag suggests that it does not need Tiller access to work, however when using it with `--dry-run` I see this error:

```
2019-05-13 14:41:27 root[16325]    ERROR Command Failed with output below:
STDOUT:
STDERR: Error: Get http://localhost:8080/api/v1/namespaces/kube-system/pods?labelSelector=app%3Dhelm%2Cname%3Dtiller: dial tcp [::1]:8080: connect: connection refused

COMMAND: helm upgrade --install metrics-server stable/metrics-server --namespace=metrics-server --dry-run --debug --version=2.4.0 --set=args[0]=--kubelet-insecure-tls --set=args[1]=--kubelet-preferred-address-types=InternalIP --set=args[2]=--metric-resolution=30s --set=args[3]=--v=3
```

**Comment by endzyme** [comment]
I think I need to get the context of original intent for `--local-development`. The README sounds like it's a flag for people developing on Reckoner itself to be able to run through logic without running real commands on helm. If that's the case then I think maybe we should evaluate if it's a commonly used feature. 

If it's to test your course file or provide some other value, then maybe the flag should be a validate type flag instead. 

Any insight you may have @ejether or @sudermanjr would be awesome if you've got some cycles.

**Comment by ejether** [comment]
The flag was there so that I could hack on it when is on an airplane and my minikube had broken and I didn't have a cluster to work from. I wanted it to, essentially, output the generated helm command.
It was intended to not need access to tiller but after the refactor and as the code path got a little more complicated, wrapping so many different aspects in differnet methods got complicated and things were missed. 

I don't have a strong opinion any more if this is a useful flag to maintain but if you look at it as a slightly broken, or leaky feature flag, it might make clarify it's intent.

**Comment by sudermanjr** [comment]
Once helm3 becomes default, this will be a thing of the past.   I vote we just leave it the way it is or remove it.

**Comment by endzyme** [comment]
After some discussion today, I think we'll use this issue to pull out the flag. Thanks again for all your help on with this info

**Comment by endzyme** [comment]
resolved in #112 and #113
